coding-test/11일차/오픈채팅방.js

```diff
@@ -0,0 +1,50 @@
+function solution(record) {
+    //로직구성
+    //0. 받아온 문자열을 쪼개서 배열로 만든다.
+    //1. 객체로 새로 들어온 아이디와, 닉네임을 매치
+    //2. Enter, Leave, Change 에 따라서 닉네밍임이 변경되는 함수로 구현
+    //3. 
+    
+    const chatRoom = {};
+    const msg = [];
```

Comment:
```suggestion
    const messageList = [];
```
약어를 사용하면 의도를 파악하기 힘든 경우가 생겨서 변수명이 조금 길어지더라도 명확한 네이밍을 짓도록 습관을 들이시는게 좋을것같아요

Reply:
명확한 네이밍 확인했습니다.